src/Symfony/Component/DomCrawler/Image.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DomCrawler;
+
+/**
+ * Image represents an HTML image (an HTML img tag).
+ */
+class Image extends AbstractUriElement
+{
+    public function __construct(\DOMElement $node, $currentUri)
+    {
+        parent::__construct($node, $currentUri);
```

Comment:
@valeriangalliat yes, I think it would be more clear. I'm not gonna block this PR if you don't do this though ;)